src/lib.rs

```diff
@@ -71,6 +71,29 @@ pub struct Args {
     /// Enables more verbose output.
     #[structopt(long, short)]
     pub verbose: bool,
+
+    /// Specify the VCS used to initialize the generated template.
+    #[structopt(long, default_value = "git")]
+    pub vcs: VCS,
+}
+
+//
+#[derive(Debug, StructOpt)]
+pub enum VCS {
```

Comment:
Just a little nit, could you please rename this enum to the regular [type conventions](https://rust-lang.github.io/api-guidelines/naming.html) that is `UpperCamelCase` so `Vcs`